src/ckzg.c

```diff
@@ -17,4 +17,5 @@
 #include "common.c"
 #include "eip4844.c"
 #include "eip7594.c"
+#include "fft.c"
```

Comment:
Hm. Is this needed? Do we need to expose `fft.c`? Does it hold any public methods?

Reply:
Yes this is needed. Otherwise there is no fft implementation. We just don't want to include this in the header file.

Without it we get these errors:

<img width="659" alt="image" src="https://github.com/user-attachments/assets/71586931-e59e-4a31-a9ce-7849f0e4a46a">